src/esn.calendar.libs/app/services/dav-request.js

```diff
@@ -1,46 +1,38 @@
 require('../app.constants.js');
 
-(function(angular) {
-  'use strict';
+angular.module('esn.calendar.libs')
+  .factory('calDavRequest', calDavRequest);
 
-  angular.module('esn.calendar.libs')
-    .factory('calDavRequest', calDavRequest);
+function calDavRequest($http, $q, tokenAPI, calCalDAVURLService, CAL_GRACE_DELAY_IS_ACTIVE) {
+  let davServerUrlPromise = null;
 
-  function calDavRequest($http, $q, httpConfigurer, CAL_DAV_PATH, CAL_GRACE_DELAY_IS_ACTIVE) {
-    return request;
+  return (method, path, headers, body, params) => _configureRequest(method, path, headers, body, params).then($http);
 
-    ////////////
-
-    function request(method, path, headers, body, params) {
-      return _configureRequest(method, path, headers, body, params).then($http);
-    }
-
-    function _ensurePathToProxy(path) {
-      return path.substring(path.indexOf('/calendars'), path.length);
+  function _configureRequest(method, path, headers = {}, body, params) {
+    if (!CAL_GRACE_DELAY_IS_ACTIVE && params) {
+      delete params.graceperiod;
     }
 
-    function _configureRequest(method, path, headers, body, params) {
-      var url = CAL_DAV_PATH;
-
-      if (!CAL_GRACE_DELAY_IS_ACTIVE) {
-        params && delete params.graceperiod;
-      }
-
-      headers = headers || {};
-
-      var config = {
-        url: httpConfigurer.getUrl(url + _ensurePathToProxy(path)),
-        method: method,
-        headers: headers,
-        params: params
-      };
+    return $q.all([tokenAPI.getNewToken(), _getDavServerUrl()])
```

Comment:
I think we need to look at this:

https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/pull/151/

Reply:
OK it is already done there, we don't need to do anything here.